Modules/Loadable/Transforms/Resources/UI/qSlicerTransformsModuleWidget.ui

```diff
@@ -170,6 +176,57 @@
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="ctkCollapsibleGroupBox" name="CenterOfTransformationGroupBox">
```

Comment:
It would be nicer to use a coordinate widget and set quantity to length (so that we get length unit). I think this is how Volumes module / Volume Information / Image Origin displays the point coordinates.